### ISSUE TITLE
Allow Users To Change Quantity In Cart

### ISSUE THREAD

**Issue by hfmw** [reactioncommerce/reaction-feature-requests#44]
It would be great if users were able to change the quantity they have in their cart without having to delete the item, and go back to the product page. I would think this functionality would be a useful addition in the drop-down cart shown below, but also once at checkout?

<img width="533" alt="screen shot 2018-04-29 at 11 44 10" src="https://user-images.githubusercontent.com/33007279/39405761-e135701e-4ba2-11e8-9521-c264ed6bed85.png">

**Comment by loanlaux** [comment]
Absolutely. I've had clients asking me for this and I think it should be built-in.

**Comment by janus-reith** [comment]
I've built such a feature in one of my plugins.
It replaces the Cartitem and relocates the quantity badge to be under the product name,
and has buttons to de/increase the quantity:

![image](https://user-images.githubusercontent.com/1831220/44982218-709a6080-af75-11e8-8bd6-e6644b77934c.png)

It also replaces the 4th checkout step to display the cart items of the cart overlay there aswell.
Will see if I can submit a PR to integrate this into core.

**Comment by hfmw** [comment]
Hi @janus-reith - that sounds perfect! It would be great if you could submit a PR for this.

Hopefully these features will be included in 2.0 - but it looks like that is still a few months away.
